IdentityCore/src/workplacejoin/ios/MSIDWorkPlaceJoinUtil.m

```diff
@@ -29,26 +29,30 @@
 #import "MSIDWorkplaceJoinChallenge.h"
 #import "MSIDWorkPlaceJoinUtilBase+Internal.h"
 
+static NSString *kWPJPrivateKeyIdentifier = @"com.microsoft.workplacejoin.privatekey\0";
+
 @implementation MSIDWorkPlaceJoinUtil
 
-+ (MSIDAssymetricKeyPairWithCert *)getWPJKeysWithContext:(id<MSIDRequestContext>)context
++ (nullable MSIDAssymetricKeyPairWithCert *)getWPJKeysWithContext:(nullable id<MSIDRequestContext>)context
+{
+    return [self getWPJKeysWithTenantId:nil context:context];
+}
+
++ (MSIDAssymetricKeyPairWithCert *)getWPJKeysWithTenantId:(NSString *)tenantId context:(id<MSIDRequestContext>)context
 {
-    return [self getRegistrationInformation:context workplacejoinChallenge:nil];
+    return [self getRegistrationInformation:context tenantId:tenantId workplacejoinChallenge:nil];
 }
 
 + (MSIDRegistrationInformation *)getRegistrationInformation:(id<MSIDRequestContext>)context
```

Comment:
as MSIDRegistrationInformation inherit MSIDAssymetricKeyPairWithCert inherit MSIDAssymetricKeyPair, and has property 
```
@property (nonatomic, nullable, readonly) NSString *keyExponent;
@property (nonatomic, nullable, readonly) NSString *keyModulus;
```

I assume this will not work for ECC key.

Reply:
Addressed in an alternative solution PR: https://github.com/AzureAD/microsoft-authentication-library-common-for-objc/pull/1083